app/views/devise/mailer/password_change.html.haml

```diff
@@ -0,0 +1,3 @@
+%p
+  Hello #{@resource.email}!
+%p We're contacting you to notify you that your password has been changed.
```

Comment:
Go through all haml files and take a look if they looks fine with latest comments